### DISCUSSION TITLE
Flickering Gnome Shell on Wallpaper Change

### DISCUSSION THREAD

**Discussion by freddy1301** [aunetx/blur-my-shell#201] [Ideas]
Hi, 

I am using Blur my Shell Extension since I got to know it and really like it.
But recently I installed variety (wallpaper changer) on my PC and the flashing of my gnome shell when the wallpaper changes every minute (because of the variety clock) is very annoying.

I would be really happy when this gets fixed :D

**Comment by aunetx** [comment]
Hello, I transferred the issue to #204!
